chapter_14_simple_form.asciidoc

```diff
@@ -536,13 +542,15 @@ A quick `grep` shows us there are several places where we're using `id_new_item`
 $ <strong>grep id_new_item src/functional_tests/test*</strong>
 src/functional_tests/test_layout_and_styling.py:        inputbox =
 self.browser.find_element(By.ID, "id_new_item")
-src/functional_tests/test_layout_and_styling.py:        inputbox =
+src/functional_tests/test_layout_and_styling.py:        inputbo§§x =
 self.browser.find_element(By.ID, "id_new_item")
 src/functional_tests/test_list_item_validation.py:
 self.browser.find_element(By.ID, "id_new_item").send_keys(Keys.ENTER)
 [...]
 ----
 
+// SEBASTIAN: Are those "§§" not a mistake here? 3rd line of grep result
```

Comment:
They definitely look like a mistake, but your commit introduced them. I was wondering if it was accidental until I saw your comment. I'm removing this change,